docs/operator-guide/quick-start.md

```diff
@@ -104,3 +104,51 @@ It is not recommended to run `:latest` for any prolonged period of time, all of
     - `:vX.Y.Z` - Specific release tag which does not move eg `:v0.0.0`
     - `:vX.Y` - Moving tag for the latest major.minor release eg `:v7.0`
     - `:vX` - Moving tag for the latest major release eg `:v7`
+
+### Creating a configuration from the tiff files
+
+Using the [basemaps CLI](https://github.com/linz/basemaps/tree/master/packages/cli#usage----bundle) a configuration bundle can be generated.
+
+A [tileset](../configuration.md) needs to be created as the entry point for the configuration.
+
+Below is a configuration for the two example layers, where the rural imagery sets underneath the urban imagery, and the urban imagery is only turned on when the zoom is greater than z15.
+
+storing this JSON into `config/aerial.json`
+
+```json
+{
+  "type": "raster",
+  "id": "ts_basic",
+  "title": "Basic Imagery Basemap",
+  "layers": [
+    {
+      // Source location for EPSG:2193 Imagery, has to be absolute
+      "2193": "/home/user/basemaps-quick-start/imagery/otago_2017-2019_0.3m/",
+      // url/slug friendly name of the imagery set
+      "name": "otago_2017-2019_0.3m",
```

Comment:
I think we not using only hyphen for the image names to be url friendly

Reply:
I think both `name` and `title` sholuld be optional, and fetched from the STAC, they should only be used if they are a override.

this specific name is only there because its "required"